bin/migrate-oats-data/noi/sql/notice_of_intent_submission/soil_fields/notice_of_intent_soil_fields_update.sql

```diff
@@ -0,0 +1,9 @@
+SELECT
+    nois.fill_project_duration_unit,
+    nois.fill_project_duration_amount,
+    nois.file_number,
+    nois.file_number::INTEGER AS file_id
```

Comment:
please use bigint for nois.file_number since since it is int8 for oats.alr_application_id